eciflow/values.yaml

```
@@ -15,8 +15,10 @@ airflow:
 
   namespace: eciflow
   serviceAccount:
-    # If not set, a name is generated using the fullname template
-    name:
+    # If present, master/worker pods will be attached to this IAM role
+    # eg. arn:aws:iam::AWS_ACCOUNT_ID:role/IAM_ROLE_NAME
+    masterIamRoleArn:
```

this way is better always in helm values 
```suggestion
    masterIamRoleArn: {}
```